lib/cylc/gui/dot_maker.py

```diff
@@ -135,114 +153,156 @@
 
 live = {
     'small': [
-        "11 11 3 1",
-        ".	c <FILL>",
-        "*	c <BRDR>",
-        "w  c <FAMILY>",
-        "***********",
-        "***********",
-        "**.......**",
-        "**.wwww..**",
-        "**.w.....**",
-        "**.www...**",
-        "**.w.....**",
-        "**.w.....**",
-        "**.......**",
-        "***********",
-        "***********"
+        "16 16 4 1",
+        ". c <FILL>",
+        "* c <BRDR>",
+        "w c <FAMILY>",
+        "x c None",
+        "xxxxxxxxxxxxxxxx",
+        "xxxxxxxxxxxxxxxx",
+        "xxxxxxxxxxxxxxxx",
+        "xxxxxxxxxxxxxxxx",
+        "xxxxxxxxxxxxxxxx",
+        "xxxxx***********",
+        "xxxxx***********",
+        "xxxxx**.......**",
+        "xxxxx**.wwww..**",
+        "xxxxx**.w.....**",
+        "xxxxx**.www...**",
+        "xxxxx**.w.....**",
+        "xxxxx**.w.....**",
+        "xxxxx**.......**",
+        "xxxxx***********",
+        "xxxxx***********"
```

Comment:
This simply adds some blank space left and top, to make room for the new indicator tags.